File: week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    // Checking if height is between 1 and 8, otherwise question repeats.
    do
    {
        height = get_int("Enter a height: ");
    }
    while (((height < 1) || (height > 8)));

    // For every row, do following
    for (int row = 1; row <= height; row++)
    {
        // Add blank spaces
        for (int blank = 1; blank <= (height - row); blank++)
        {
            printf(" ");
        }
        // Add first part of the pyramid
        for (int hash1 = 1; hash1 <= row ; hash1++)
        {
            printf("#");
        }
        // Add two blank spaces to move second part of the pyramid further.
        printf(" ");
        printf(" ");

        // Add the second part of the row
        for (int hash2 = 1; hash2 <= row; hash2++)
        {
            printf("#");
        }

        // After the row is finished, go to the newline.
        printf("\n");
    }
}

File: week6/sentimental-mario-more/mario.py
# TODO

while True:
    try:
        height = int(input("Enter a height: "))
    except ValueError:
        print("That's not an integer.")
    else:
        if height > 0 and height < 9:
            break

# For every row
for row in range(1, height + 1):
    # Add blank spaces
    for blank in range(height-row):
        print(" ", end="")
    # Print first part of pyramid
    for prt1 in range(row):
        print("#", end="")
    # Print two blank spaces
    print("  ", end="")
    # Print second part of pyramid
    for prt1 in range(row):
        print("#", end="")
    # After row is finished, go to newline
    print("")
